features/FxA-54-connect-another-device-phase-3/README.md

```diff
@@ -0,0 +1,364 @@
+4. Open the SMS link on an iOS device. The link will be of the form `https://<server>/m/:signinCode`, write down the `signinCode`.
+5. Safari will open and a dialog will ask `Open this page in "App Store"?` Touch `Open`.
+6. The Apple app store will open to the Firefox page. Click the install button.
+7. Firefox should install. When complete, click `Open`.
+8. Firefox will open to the Firefox for iOS firstrun page. Click `Start Browsing`.
+9. Open a tab to `https://<server>/signin?context=fx_ios_v1&service=sync&signin=<signinCode from step 4>`
+10. When opening Firefox Accounts, email address from step 1 should be filled in.
+11. Sign in with the password from step 1.
+12. The user will be unable to sign in, even if entering the correct password from
+   step 1 because Firefox for iOS will not be listening for messages from FxA.
+
+###### Sign up in Firefox desktop, send link to iOS, Firefox already installed
+
+1. Using Firefox desktop, sign in to Sync by creating a new Firefox Account using
+   a mozilla or softvision address.
+2. In the same instance of Firefox desktop, open the verification link with
+   the `&signinCodes=true&forceExperimentGroup=treatment` query parameters.
+3. Enter a valid phone number for an iOS device, submit.
+4. Open the SMS link on an iOS device. The link will be of the form `https://<server>/m/:signinCode`, write down the `signinCode`.
+5. Safari will open and a dialog will ask `Open this page in "App Store"?` Touch `Open`.
+6. The Apple app store will open to the Firefox page. Click `Open`.
+7. Firefox should open.
+8. Open a tab to `https://<server>/signin?context=fx_ios_v1&service=sync&signin=<signinCode from step 4>`
+9. When opening Firefox Accounts, email address from step 1 should be filled in.
+10. The user will be unable to sign in, even if entering the correct password from
+   step 1 because Firefox for iOS will not be listening for messages from FxA.
+
+##### Testing with support for deep links in Firefox for iOS
+
+###### Sign up in Firefox desktop, send link to iOS, Firefox not installed
+
+1. Using Firefox desktop, sign in to Sync by creating a new Firefox Account using
+   a mozilla or softvision address.
+2. In the same instance of Firefox desktop, open the verification link with
+   the `&signinCodes=true&forceExperimentGroup=treatment` query parameters.
+3. Enter a valid phone number for an iOS device, submit.
+4. Open the SMS link on an iOS device.
+5. Safari will open and a dialog will ask `Open this page in "App Store"?` Touch `Open`.
+6. The Apple app store will open to the Firefox page. Click the install button.
+7. Firefox should install. When complete, click `Open`.
+8. Firefox will open to the Firefox for iOS firstrun page. Complete the firstrun flow.
+9. Firefox Accounts should open, email address from step 1 should be filled in.
+10. Sign in with the password from step 1.
+
+###### Sign up in Firefox desktop, send link to iOS, Firefox already installed
```

Comment:
Assuming that using updated Firefox iOS (support deep and universal links into FxA), this should not redirect to App Store, but rather open app directly. Should be ok to remove step 6 here.